tests/parser/features/test_logging.py

```diff
@@ -107,7 +107,11 @@ def test_event_logging_cannot_have_more_than_three_topics(assert_tx_failed):
     loggy_code = """
 MyLog: __log__({arg1: indexed(bytes <= 3), arg2: indexed(bytes <= 4), arg3: indexed(address), arg4: indexed(num)})
 
+Transfer: __log__({_from: indexed(num), _to: indexed(num), _value: num})
+
 @public
+def foo():
+    log.Transfer(1, 2, 3, 4, 5, 6, 7, 8, 9 ,10)@public
 def foo():
     log.MyLog('bar', 'home', self)
```

Comment:
If `foo()` defined twice here? How does this work? Also `@public` for the second `foo()` is messed up